libs/javalib/worker/src/mill/javalib/zinc/ZincWorker.scala

```diff
@@ -106,9 +106,10 @@ class ZincWorker(
           loaderLibraryOnly = ClasspathUtil.rootLoader,
           libraryJars = Array(libraryJarNameGrep(
             compilerClasspath,
-            // we don't support too outdated dotty versions
-            // and because there will be no scala 2.14, so hardcode "2.13." here is acceptable
-            if (JvmWorkerUtil.isDottyOrScala3(key.scalaVersion)) "2.13." else key.scalaVersion
+            // if Dotty or Scala 3.0 - 3.7, use the 2.13 version of the standard library
+            if (JvmWorkerUtil.enforceScala213Library(key.scalaVersion)) "2.13."
```

Comment:
So Scala 2.12 will not enter this branch right?

Reply:
Exactly.